edpm_ansible/ansible_plugins/modules/container_config_hash.py

```diff
@@ -145,15 +139,45 @@ def _update_container_config(self, path, config):
         os.chmod(path, 0o600)
         self.results['changed'] = True
 
-    def _get_config_hash(self, config_volume):
-        """Returns a config hash from a config_volume.
+    def _read_file(self, file):
+        """Read a given file and return its content
+        :param file: string
+        :returns: bytes
+        """
+
+        r = b''
+        with open(file, 'rb') as f:
+            while True:
+                data = f.read(BUF_SIZE)
+                r += data
+                if not data:
+                    break
+        return r
+
+    def _create_checksum(self, config_volume, exclusions=[]):
```

Comment:
IMO create indicates this creates a resource. Probably get or calculate would be better.

Reply:
ack